licenses.d2iq.yaml

```
@@ -232,6 +232,14 @@ resources:
     sources:
       - ref: ${image_tag}
         url: https://github.com/mesosphere/kubetunnel
+  - container_image: docker.io/mesosphere/nkp-pulse-management:v0.0.1-dev.0
```

Nothing blocking this but, do we want to release as v0.1.0 for the first version? We did so for Insights.
So then we can have v0.1.0-dev.X releases till we cut the first RC for NKP v2.14
```suggestion
  - container_image: docker.io/mesosphere/nkp-pulse-management:v0.1.0-dev.0
```

---

Apparently leaving comments blocks merging, which I didn't know. Resolving.